app/controllers/companies_controller.rb

```diff
@@ -13,4 +15,14 @@ def show
     end
   end
 
+  private
```

Comment:
Just to follow Josh's advice, I would add another indentation on methods under private